docs/chapters/ABOUT_ONEKO.md

```diff
@@ -0,0 +1,79 @@
+
+O-Neko is a Kubernetes-native application that allows to cross a bridge between developers and other stakeholders by
+deploying development versions of your software into Kubernetes to allow everybody to try and test them.
+
+## Features
+
+The most important features of O-Neko are:
+
+* Deploy development versions (e.g. every branch) to Kubernetes with one click
+* Configure projects with native Kubernetes .yaml files
+* Automatically re-deploy running versions when the corresponding Docker image has changed (configurable per project and version)
+* Automatically stop running versions after a specific time (configurable per project and version)
+* Project meshes that allow you to wrap multiple projects into one pool and deploy specific versions of them together,
+effectively providing a way to deploy dynamic test server setups with ease
+* Select variables that allow fast changes to frequently used configuration settings (e.g. server URLs)
+
+## Pre-requisites
+
+Generally, nearly every project able to run in Kubernetes can be deployed with O-Neko. To run a project in O-Neko:
+
+* You need Docker containers of all project versions you want to deploy with O-Neko
+* You need to provide the Kubernetes .yaml files in a simple template format to configure a project
+* O-Neko works with kubernetes versions 1.9.0 - 1.15.3
+
+## How does it work?
+
+* O-Neko is running inside your Kubernetes cluster
+* A project consists of a Docker image that resides in any Docker registry. O-Neko is polling all tags that are available for this image and lets you deploy them.
+* The configuration is done with native Kubernetes .yaml and template variables. O-Neko provides some variables (e.g. the docker image tag), but you can also define your own. The configuration and the variables can be overridden by specific versions.
+* When a version of a project is deployed, O-Neko creates a namespace in Kubernetes to deploy all resources to. If you stop a deployment, the corresponding namespace will be deleted.
+* The status of all running deployments is monitored by O-Neko and can be seen on the web frontend
+
+## O-Neko vocabulary
+
+In O-Neko as well as in this documentation we often use terms that may be misunderstood for something else. We want
+to clarify what specific words mean in the context of O-Neko.
+
+### Project
+A project is the entry point for deploying an application with O-Neko. Projects have a direct
+association with a Docker image. Theoretically you can also have multiple projects referencing the same Docker image.
+A project can have multiple versions. Projects can be configured using configuration templates.
+
+### Project Version
+
+A project version is associated with a tag of the project's docker image.
+A project version inherits the project's configuration templates but can also override them or add entirely new configuration
+templates. A project version can be deployed and automatically updated if the docker tag gets pushed again with different
```

Comment:
We might add here, that this is intended for scenarios where your docker tags are of a rather snapshot-ish nature.

Reply:
I phrased it a bit differently in the hopes that it's even clearer to understand.

Reply:
Sure, I did not intend to use this wording, but just throw in the idea generally.